about.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <script
+      src="https://kit.fontawesome.com/7d39ffa61e.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>REFUGEE FOOD FESTIVAL</title>
+  </head>
+  <body class="about-body">
+    <header class="d-flex j-content-btw nav-bar">
+      <div class="name-div">
+        <a href="index.html" class="name-anchor">
+          <img
+            class="project-logo"
+            src="./images/restuarant.logo.png"
+            alt="page logo"
+        /></a>
+      </div>
+      <nav class="navigation-link d-flex j-content-btw">
+        <button
+          type="button"
+          class="toggle-button justify-content-space-between display-none-lg"
+          aria-label="toggle-button"
+        >
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
+
+        <button class="x-button display-none">
+          <span> &#10005;</span>
+        </button>
+
+        <ul class="display-none display-flex-lg justify-con-btw nav-ul">
+          <li class="nav-ul-li">
+          <li class="nav-ul-li"><a href="" class="nav-li-anchor">About</a></li>
+          <li class="nav-ul-li">
+            <a href="" class="nav-li-anchor">Program</a>
+          </li>
+          <li class="nav-ul-li"><a href="" class="nav-li-anchor">Join</a></li>
+          <li class="nav-ul-li">
+            <a href="" class="nav-li-anchor">Sponsors</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="text-align">
+      <section class="home-Page">
+        <div class="home-page-first-div">
+          <p class="home-page-title">"Refugee Food Festival"</p>
+          <h1 class="home-page-header">
+            A Blue Pacific voice for the United Nations Food Systems Summit
+          </h1>
+          <p class="about-paragraph text-align-center">
+            Youth in refugee situations from around the world came together to
+            discuss their views, perspectives, and ideas on how the UN Food
+            Systems Summit activities and efforts to transform food systems can
+            be leveraged for greater food security, better nutrition, and more
+            viable livelihoods for refugees.
+          </p>
+
+          <p class="about-second-paragraph text-align-center">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam
+            elementum risus
+          </p>
+        </div>

```

Comment:
- [ ] Please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and implement the required changes on your main section on both desktop and mobile version
![3](https://user-images.githubusercontent.com/47625407/157781680-d5dc993a-f5ca-4ab7-a22e-7ccfd2d2c153.png)